source/crud/read-operations/retrieve.txt

```diff
@@ -136,8 +166,21 @@ purchased color.
 For more information about how to construct an aggregation pipeline, see
 the {+mdb-server+} manual page on :manual:`Aggregation </aggregation>`.
 
-For additional information on the methods mentioned on this page, see
-the following API Documentation:
+API Documentation
+-----------------
+
+For more information on the methods and classes used to retrieve documents
+on this page, see the following API Documentation:
+
+- `find() <{+api+}/apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#find()>`__
+- `first() <{+api+}/apidocs/mongodb-driver-sync/com/mongodb/client/MongoIterable.html#first()>`__
+- `FindIterable <{+api+}/apidocs/mongodb-driver-sync/com/mongodb/client/FindIterable.html>`__
+- `aggregate() <{+api+}/apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#aggregate(java.util.List)>`__
+
+Server Manual Entries
+---------------------
```

Comment:
S: outside the scope of this ticket so up to you, but I'd put this section and API Documentation as subsections under "Additional Information"

Reply:
applies to all